windows/drivers/uwb/cx/include/windows/devices/uwb/UwbCxDdiLrp.hxx

```diff
@@ -9,4 +9,10 @@
 
 #include <windows/devices/uwb/UwbCxLrpDevice.h>
 
+namespace windows::drivers::uwbcx
+{
+const std::string UwbNotificationNamespace = "\\Notifications";
```

Comment:
You should be able to use `costexpr auto` here, which will allow this being used in a wider ranger of string contexts (`std::string`, `std::string_view`, `char*`, `std::array<char>`, etc.).

Reply:
It wouldn't compile for some reason when I tried that

Reply:
Odd, I gave it a try and it worked. Oh well.